### DISCUSSION TITLE
Add Bulk Document Imports

### DISCUSSION THREAD

**Discussion by FUsIon80** [astubenbord/paperless-mobile#223] [Feature Requests]
I don't found this Feature, so here I write ^^

I would be nice if I could Import many Dokuments at bulk... Or is there a folder at Filesystem Where I could Sent my files to Import?

**Comment by astubenbord** [comment]
Hi, so bulk imports are not currently supported as paperless does not expose such an endpoint. 
The consumption mechanism works well on the paperless host since paperless can directly access this folder and does not require an API to consume these files individually. 

Anyway, it would _theoretically_ be possible to select multiple files and let the app upload them individually (i.e. add all files to a queue and for each file, an individual API request will be performed one after another or even all at the same time). But keep in mind that if you're trying to upload 100s of files with 1 MB each, you'll have to upload 100s of MB which could take a while, and then there's also the question what happens when an upload fails (even after retrying). I think the API endpoints are not as robust as the consumption through the consume folder. But it may be worth a try.

**Comment by FUsIon80** [comment]
For me it would be nice ;)

**Reply by astubenbord** [reply]
I've updated the title so I'll find it more easily in the future :)
